### DISCUSSION TITLE
增加字幕搜索，并一键跳转至时间点的功能。

### DISCUSSION THREAD

**Discussion by BR184** [the1812/Bilibili-Evolved#4707] [功能建议 / Ideas]
回看一些超长录播时，比如极客湾录播，其实就想听视频里的几个观点。

但是每次这种录播一般UP都是几个小时连续谈论不同话题，寻找你想了解的话题如大海捞针。

不过B站拥有**AI字幕**功能后，可以借助**下载字幕**功能，打开下载好的字幕后**搜索关键字**，再将关键字对应的时间戳**转换为视频时间**点，最后在网页视频**手动跳转**到目标来观看想看的内容。

所以我希望能增加一个**全自动**帮你完成这件事儿的功能，来**方便跳转**到超长录播中对应的内容。

**Comment by Geeyun-JY3** [comment]
在 #3867 的基础上再加个搜索？

**Reply by BR184** [reply]
我觉得不错
